charts/spire/charts/spire-server/templates/statefulset.yaml

```diff
@@ -137,6 +137,45 @@ spec:
               mountPath: /tmp
               readOnly: false
         {{- end }}
+
```

Comment:
up at the very top of the file, should have:
a configmap checksum and annotation so that it updates on value chage.